test.sh

```diff
@@ -16,6 +16,34 @@ function main(){
     else
         run_production_tests;
     fi
+
+    # FIXME: currently only work on development settings
+    if [[ $(ls -l docker-compose.yml) == *"development.yml"* ]]; then
+        if [ $RUN_E2E ]; then
+            if [ $RUN_E2E = 'true' ]; then
```

Comment:
Lazy evaluation should be possible with

    if [[ $(ls -l docker-compose.yml) == *"development.yml"* ]] && [ $RUN_E2E ] && [ $RUN_E2E = 'true' ]; then
if I understand http://wiki.bash-hackers.org/commands/classictest#why_you_should_avoid_using_-a_and_-o correctly.